### ISSUE TITLE
Enable telly to parse m3u without using the sed command

### ISSUE THREAD

**Issue by tombowditch** [tombowditch/telly#35]
Should be fairly easy to do. The m3u parser we're using is MIT license so we can fork off of that.  I'll work on a proposal later.

**Comment by tombowditch** [comment]
Progress begins: https://github.com/tombowditch/telly-m3u-parser

**Comment by tombowditch** [comment]
Great news for all...


```
[Toms-MacBook-Pro ~/code/telly] go run main.go -playlist http://my provider/x\?username\=x\&password\=x
[telly] [info] Downloading file http://x/x?username=x&password=x
[telly] [info] Reading m3u file /var/folders/_f/x/T/telly.m3u...
[telly] [warning] telly is not attempting to strip out unneeded channels, please use the flag -filterregex if telly returns too many channels
[telly] [info] telly is currently not filtering for only uk television. if you would like it to, please use the flag -uktv
[telly] [info] found 9697 channels             <-- this is the good thing
[telly] [info] creating discovery data
[telly] [info] creating lineup status
[telly] [info] creating device xml
[telly] [info] creating webserver routes
[telly] [info] Building lineup
[telly] [info] listening on localhost:6077            <-- it boots, no sed needed.
```

Expect an update soon.

T

**Comment by tombowditch** [comment]
Need a couple testers if anyone's available

**Comment by tombowditch** [comment]
fixed in v0.4.1
